package.json

```diff
@@ -76,7 +76,8 @@
         "typescript": "4.7.4",
         "react-native": "0.68.2",
         "prettier": "2.7.1",
-        "type-fest": "2.12.2"
+        "type-fest": "2.12.2",
+        "bcrypto": "5.4.0"
```

Comment:
Why is it in root `package.json` and not in `@trezor/coinjoin` package?

Reply:
I guess because it's not direct dependency so we need to force it here in `resolutions`?